app/components/provider_interface/diversity_information_component.rb

```diff
@@ -40,11 +40,12 @@ def offer_context
     end
 
     def disability_value
-      disabilities = equality_and_diversity['disabilities'].map do |disability|
+      disabilities = Array(equality_and_diversity.fetch('disabilities', []))
```

Comment:
Do we need the fetch? The Array method already does this (or the other way around) :)

Reply:
Updated in afb44e3b